kubemarine/procedures/check_iaas.py

```diff
@@ -36,7 +36,7 @@
 from kubemarine.core.group import NodeGroupResult
 
 def connection_ssh_connectivity(cluster):
-    with TestCase(cluster.context['testsuite'], '001', 'SSH', 'Connectivity', default_results='Connected'):
+    with TestCase(cluster.context['testsuite'], cluster.context['execution_arguments'], '001', 'SSH', 'Connectivity', default_results='Connected'):
```

Comment:
Maybe we could pass the only property cluster.context to TestCase constructor, and fetch both properties `testsuite`, 'execution_arguments' from it there.

Reply:
Done, passed `cluster` to TestCase to take `cluster.log` too